### DISCUSSION TITLE
提取关键信息疑问

### DISCUSSION THREAD

**Discussion by DooJay** [PaddlePaddle/PaddleOCR#14182] [Q&A]
![image](https://github.com/user-attachments/assets/356f69e9-0799-40b5-b94b-e6286b5ce5af)
这是我的标注文件，我只想提取这些关键信息，这几个信息都标注成了不同的分类
模型训练后得到hmean=1

但是预测模型后不想提取的文本也被标注了定义的几个类
使用类似的命令
`python3 tools/infer_kie_token_ser.py -c fapiao/ser_vi_layoutxlm.yml -o Architecture.Backbone.checkpoints=fapiao/models/ser_vi_layoutxlm_fapiao_udml/best_accuracy Global.infer_img=./train_data/zzsfp/imgs/b25.jpg Global.infer_mode=True`

得到如下结果
![b23_ser](https://github.com/user-attachments/assets/16da596c-7d27-447c-8e89-5e6000b4a096)

**我只想到得到类似标注的结果，其他未标注的全部忽略，应该怎么做啊?**

**Comment by star20245** [comment]
你可以先运行为训练前的DET检测模型，看下原生识别的结果。
对比你训练的det模型后 单张图片的det识别效果,是不是只识别了你标注的部分，如果不是那说明训练数据没有达到预期。
或者你可以换种思路，除了除了你需要的信息加上标注，其他的也要标注只是都标注成other，看是否会提高你的识别率

**Reply by DooJay** [reply]
det需要自己训练吗，如何训练啊，只标注图上需要识别的文本框，不需要的不标注这样吗
